util.c

```diff
@@ -93,6 +97,192 @@ get_user_cfgfile_path(const char *common_path, const char *filename, const struc
   return 1;
 }
 
+/*
+ * This function will look for users name with valid user token id, in a database Mysql
+ *
+ * Returns one of AUTH_FOUND, AUTH_NOT_FOUND, AUTH_NO_TOKENS, AUTH_ERROR.
+ *
+ * Need database with this table structure : 
+ * 
+ * CREATE TABLE IF NOT EXISTS  `otp`.`yubikey_mappings` ( 
+ *   `otp_id` VARCHAR(12) NOT NULL ,  
+ *   `username` VARCHAR(64) NOT NULL ,     
+ *   PRIMARY KEY  (`otp_id`(12))
+ *  );
+ *
+ */
+int
+check_user_token_mysql (const char *mysql_server,
```

Comment:
There's some trailing whitespace in this function, please get rid of those.

Some if statements don't have {} please add that.

Reply:
Also the "if(verbose)" ?

Reply:
not important, everything is already a mess.. =)

Reply:
there is still trailing whitespace.